src/otx/algorithms/detection/adapters/mmdet/models/detectors/custom_lite_dino.py

```diff
@@ -0,0 +1,21 @@
+"""OTX Lite-DINO Class for object detection."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from mmdet.models.builder import DETECTORS
+
+from otx.algorithms.common.utils.logger import get_logger
+from otx.algorithms.detection.adapters.mmdet.models.detectors import CustomDINO
+
+logger = get_logger()
+
+
+@DETECTORS.register_module()
+class CustomLiteDINO(CustomDINO):
+    """Custom Lite-DINO <https://arxiv.org/pdf/2303.07335.pdf> for object detection."""
+
+    def load_state_dict_pre_hook(self, model_classes, ckpt_classes, ckpt_dict, *args, **kwargs):
+        """Modify official lite dino version's weights before weight loading."""
+        super(CustomDINO, self).load_state_dict_pre_hook(model_classes, ckpt_classes, ckpt_dict, *args, *kwargs)
```

Comment:
Is it your intention to use `load_state_dict_pre_hook` of `CustomDeformableDETR` instead of `CustomDINO`'s?

Reply:
Why do we need to create additional model class if there is no visible changes?

Reply:
This class is needed for smart weight loading and ov export. It inherits custom patches of mmdeploy for ov export from CustomDINO. However CustomDINO has complex smart weight loading algorithm because it comes from mmdet3.x. So Lite DINO has to inherit simple smart weight logic from Deformable DETR.